test/h.test.js

```diff
@@ -67,47 +67,12 @@ test("vnode with attributes", () => {
   })
 })
 
-test("skip null and Boolean children", () => {
```

Comment:
@frenzzy Why did you remove this test? Please add it back.

Reply:
This test was moved from `h.test.js` to `components.test.js` because the check for `null`, `true` and `false` was moved from `h` to runtime.